ios/mapswipe.xcodeproj/project.pbxproj

```diff
@@ -983,7 +983,7 @@
 				CODE_SIGN_ENTITLEMENTS = mapswipe/mapswipe.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Distribution";
 				CODE_SIGN_STYLE = Manual;
-				CURRENT_PROJECT_VERSION = 5;
+				CURRENT_PROJECT_VERSION = 2;
```

Comment:
Should we be changing the project version.

Reply:
See my other note just below